packages/extension-browser/package.json

```
@@ -123,6 +123,7 @@
     "lint:dependencies": "node ../../scripts/lint-dependencies.js",
     "lint:md": "node ../../scripts/lint-markdown.js",
     "package-extension": "web-ext build -s ./dist/bundle -a ./dist --overwrite-dest && npm run lint-bundle-size",
+    "sign-extension": "web-ext sign -s ./dist/bundle",
```

Can we have this output in the `./dist` folder just like `package-extension` does (if so the `.gitignore` update can be removed)? Also does it need the `--overwrite-dest` flag as well?
```suggestion
    "sign-extension": "web-ext sign -s ./dist/bundle -a ./dist --overwrite-dest",
```

---

`--overwrite-dest` doesn't work in this case. Also this command doesn't require running `package-extension`